postgres_docker_init/src/assignment.py

```diff
@@ -0,0 +1,47 @@
+import psycopg2
+from psycopg2 import sql
+
+# Database connection parameters
+DB_NAME = "victor_akinode_db"
```

Comment:
It's bad practice to hardcode your credentials like this, please move the credentials into a .env file (ensure the file is in your gitignore too!)

Reply:
I have resolved this. Thanks.